src/summary_statistics/tmrca.cc

```diff
@@ -22,13 +22,14 @@
 #include "tmrca.h"
 
 void TMRCA::calculate(const Forest &forest) {
-  if (forest.calcSegmentLength() == 0) return;
+  //std::cout << "forest.calcSegmentLength() "<<forest.calcSegmentLength() <<std::endl;
+  if (forest.rec_bases_.size() == 1) return;
```

Comment:
Hi Paul, 

https://github.com/scrm/scrm/pull/92#discussion_r41728933, valgrind complained about for not initialized value here. Hence I made it friend to class forest.

Best
Joe